File: Python/Warmup-2/string_times.py

# this function return a new string that is n copies of the given string
# n is a non negative int value
def string_times(str, n):
	s = ""

	for _ in range(n):
		s = s + str

	return s

print(string_times("Hi", 2))
print(string_times("Hi", 3))
print(string_times("Hi", 1))

File: JavaScript/Warmup-2/stringTimes.js

// given a string and a non-negative int n, return a larger string that is n copies of the original string
const stringTimes = (str, n) => {
    let newStr = '';

    for (let i = 0; i < n; i++) {
        newStr = newStr + str;
    }

    return newStr;
};

console.log(stringTimes('Hi', 2));
console.log(stringTimes('Hi', 3));
console.log(stringTimes('Hi', 1));
